sympy/polys/solvers.py

```diff
@@ -4,6 +4,8 @@
 
 from sympy.matrices import MutableDenseMatrix, zeros
 
+from ..matrices.reductions import _rref
+
```

Comment:
-> absolute import

Reply:
Won't import as absolute and @oscarbenjamin did not want to make `_rref` public for absolute imports, only way I can do this.